tests/Integration/Behaviour/Features/Scenario/Product/SpecificPrice/edit_specific_price.feature

```diff
@@ -0,0 +1,191 @@
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+    When I edit specific price "price1" with following details:
+      | includes tax  | false |
+      | price         | 50    |
+      | from quantity | 3     |
+    Then specific price price1 should have following details:
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | false  |
+      | price                 | 50     |
+      | from quantity         | 3      |
+      | from                  |        |
+      | to                    |        |
+    When I edit specific price "price1" with following details:
+      | price         | 45.78 |
+      | includes tax  | true  |
+      | from quantity | 1     |
+    Then specific price price1 should have following details:
+      | specific price detail | value  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+
+  Scenario: I edit specific price relations
+    Given specific price price1 should have following details:
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+      | shop group            |        |
+      | shop                  |        |
+      | currency              |        |
+      | country               |        |
+      | group                 |        |
+      | customer              |        |
+    When I edit specific price "price1" with following details:
+      | shop group | 42 |
+      | shop       | 51 |
+      | currency   | 69 |
+      | country    | 21 |
+      | group      | 33 |
+      | customer   | 99 |
+    Then specific price price1 should have following details:
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+      | shop group            | 42     |
```

Comment:
I see you use random int values, Which makes me wonder if the handler shouldn't be responsible for checking if these entities exist (each one of them)
I know it makes the behat test more complex to handle, but it would probably make the command more robust